src/tiled/mainwindow.ui

```diff
@@ -591,6 +592,17 @@
     <string>Edit Commands...</string>
    </property>
   </action>
+  <action name="actionClearView">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Clear View</string>
+   </property>
+   <property name="shortcut">
+    <string>Q</string>
```

Comment:
Yeah, it was the original idea, but with the way widgets are currently positioned, main toolbar and document tabs receive keypress and can be navigated with Tab, unsure if that's intended behavior or not, so I figured it would be safer to bind it to something else rather than breaking navigation.